challenge/src/App.jsx

```diff
@@ -1,45 +1,270 @@
+                }
+                alt="RandomizedPicsum"
+                height="20%"
+                width="20%"
+                style={{
+                  marginTop: "5%",
+                  marginBottom: "5%",
+                  marginRight: "5%",
+                  marginLeft: "5%"
+                }}
+              ></img>
+            </div>
+            <div style={{ backgroundColor: "whitesmoke" }}>
+              <br />
+            </div>
+          </div>
+        );
+      case 3:
+        return (
+          <div>
+            {" "}
+            <h1 style={{ fontSize: 100, color: "#F09F70" }}>
+              <center>Contact Page</center>
+            </h1>
+            <div style={{ backgroundColor: "whitesmoke" }}>
+              <br />
+            </div>
+            <div
+              style={{
+                backgroundColor: "#546882",
+                display: "flex",
+                height: "100%",
+                alignItems: "center",
+                justifyContent: "center",
+                flexDirection: "row"
+              }}
+            >
+              <img
+                src={"https://picsum.photos/id/" + param.PageRoute * 5 + "/500"}
+                alt="RandomizedPicsum"
+                height="20%"
+                width="20%"
+                style={{
+                  marginTop: "5%",
+                  marginBottom: "5%",
+                  marginRight: "5%",
+                  marginLeft: "5%"
+                }}
+              ></img>
+              <img
+                src={
+                  "https://picsum.photos/id/" + param.PageRoute * 10 + "/500"
+                }
+                alt="RandomizedPicsum"
+                height="20%"
+                width="20%"
+                style={{
+                  marginTop: "5%",
+                  marginBottom: "5%",
+                  marginRight: "5%",
+                  marginLeft: "5%"
+                }}
+              ></img>
+              <img
+                src={
+                  "https://picsum.photos/id/" + param.PageRoute * 20 + "/500"
+                }
+                alt="RandomizedPicsum"
+                height="20%"
+                width="20%"
+                style={{
+                  marginTop: "5%",
+                  marginBottom: "5%",
+                  marginRight: "5%",
+                  marginLeft: "5%"
+                }}
+              ></img>
+            </div>
+            <div style={{ backgroundColor: "whitesmoke" }}>
+              <br />
+            </div>
+          </div>
+        );
+      default:
+        return <div>An error occured.</div>;
     }
+  }
+
+  render() {
+    console.log("User has selected page # ", this.state.currentPage);
+    //console.log("Render Method is Working. Redered successfully! I think?")
+
+    return (
+      <>
+        <Navbar
+          fixed={true}
+          items={["Home", "Sponsors", "About", "Contact us"]}
```

Comment:
the names are passed into the navbar, perhaps instead of the callback giving the index, have the callback give the page name, for a more friendly interface